KSP2Runtime/KontrolModuleExtensions.cs

```diff
@@ -35,11 +67,61 @@ public static class KontrolModuleExtensions {
             }
         }
 
+        public static EntrypointArgumentDescriptor[] GetFlightEntrypointArgumentDescriptors(this IKontrolModule module, ITO2Logger logger = null) => GetEntrypointParameterDescriptors(module, MainFlight, logger);
+        public static EntrypointArgumentDescriptor[] GetKSCEntrypointArgumentDescriptors(this IKontrolModule module, ITO2Logger logger = null) => GetEntrypointParameterDescriptors(module, MainKsc, logger);
+        public static EntrypointArgumentDescriptor[] GetTrackingEntrypointArgumentDescriptors(this IKontrolModule module, ITO2Logger logger = null) => GetEntrypointParameterDescriptors(module, MainTracking, logger);
+        public static EntrypointArgumentDescriptor[] GetEditorEntrypointArgumentDescriptors(this IKontrolModule module, ITO2Logger logger = null) => GetEntrypointParameterDescriptors(module, MainEditor, logger);
+
+        private static EntrypointArgumentDescriptor[] GetEntrypointParameterDescriptors(IKontrolModule module, string name, ITO2Logger logger = null) {
+            try {
+                IKontrolFunction function = module.FindFunction(name);
+                if (function == null || function.Parameters.Count <= 1) {
+                    throw new Exception($"Function {name} does not exist or does not have any parameters");
+                }
+
+                return function.Parameters.Skip(1).Select(param => {
+                    if (param.type.Name == "int") {
```

Comment:
It should be save to use `param.type == BuiltinType.Int` (or `BultinType.Float`, `BultinType.Bool`)
Those are supposed to be singletons